### ISSUE TITLE
Cannot run run_cc_test_windows.bat: command line too long

### ISSUE THREAD

**Issue by snnn** [tensorflow/tensorflow#15542]
------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Windows Server 2012
- **TensorFlow installed from (source or binary)**:
source
- **TensorFlow version (use command below)**:
1.4.1
- **Python version**: 
3.5.4
- **Bazel version (if compiling from source)**:
0.9.0
- **GCC/Compiler version (if compiling from source)**:
376e2cfdab31f4da251ea2e50992a9bf97fd171b
- **CUDA/cuDNN version**:
None
- **GPU model and memory**:
None
- **Exact command to reproduce**:
tensorflow/tools/ci_build/windows/cpu/bazel/run_cc_test_windows.bat

### Describe the problem
Indeed, I have some local modification.  I'm trying to fix some failed cc_tests on Windows, and turn on them in bazel_test_lib.sh 

The problem is: As the number of tests is growing,  we can't pass all their names through command line. We need a new way to filter the tests. 
How about: 
Remove the settings in tensorflow/tools/ci_build/windows/cpu/bazel/run_cc_test_windows.bat,  replace them with bazel tags like no_windows, gpu_only, ...

### Source code / logs
[build_log.txt](https://github.com/tensorflow/tensorflow/files/1577981/build_log.txt)

**Comment by rongjiecomputer** [comment]
Perhaps related to bazelbuild/bazel#4335?

**Comment by snnn** [comment]
Yes... It's that issue.

**Comment by gunan** [comment]
@meteorcloudy PTAL

**Comment by meteorcloudy** [comment]
Yes, we had the same problem on our CI, it's a limitation of Windows command line length, which is only 8191 characters...

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by gunan** [comment]
@meteorcloudy do you thin we can use any workarounds?
Do you have any recommendations?

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @meteorcloudy, @gunan: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @meteorcloudy, @gunan: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @meteorcloudy, @gunan: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @meteorcloudy, @gunan: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @meteorcloudy, @gunan: It has been 108 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by meteorcloudy** [comment]
I think we should mark all the failing cc tests as "no_windows", just like the python tests. Then we won't have the command line length limit.

We're not running cc tests on TF CI yet, so it's not my first priority currently, feel free to send a PR for this.
